doc-content/enterprise-only/getting-started/data-model-proc.adoc

```diff
@@ -1,51 +1,72 @@
 [id='_defining_a_data_model']
-= Viewing the *Mortgage_Process* data objects
+= Creating the mortgage-process data objects
 
 Data objects are the building blocks for the rule assets that you create. Data objects are custom data types implemented as Java classes in specified packages of your project. These custom data types determine what data your assets and your decision services are based on.
 
-This tutorial uses the following data objects:
+The *mortgage-process* project uses the following data objects:
```

Comment:
lc mortgage_process here and Mortgage_Process in other places.

Reply:
Changed to text for this intro.